src/components/sky-nav/sky-nav.scss

```diff
@@ -182,7 +182,13 @@ $_masthead-height-sm: ms.step(7);
   margin: (ms.step(-6) * -1); /* 2 */
   transition: opacity transition.$slow ease.$out,
     transform transition.$quick ease.$out;
-  width: ms.step(8);
+  width: fluid.fluid-calc(breakpoint.$s, breakpoint.$l, ms.step(7), ms.step(9));
```

Comment:
@tylersticka Any idea why the last number has to be 9? If I change it to 8, at the large breakpoint it is very discontinuous and there is a large jump. I tried to figure out what was going wrong but I couldn't figure it out. It might have something to do with the units?

Reply:
I'm honestly not exactly sure, the math I stole from CSS-Tricks articles. I've been considering whether long-term we should replace this logic with something simpler based on math functions like `min` and `max`, which I didn't realize until recently are [startlingly well-supported](https://caniuse.com/css-math-functions).

Can you add a comment explaining why it's set to `9` here?